libsql/internal/ws/driver.go

```diff
@@ -50,15 +54,24 @@ func (r *rows) Next(dest []driver.Value) error {
 }
 
 type conn struct {
-	ws *websocketConn
+	ws  *websocketConn
+	url string
+	jwt string
+	mu  sync.Mutex
```

Comment:
Is this mutex really needed? I think database/sql never sends more than one request to a connection at a time.